runtime/compiler/net/CommunicationStream.cpp

```diff
@@ -83,6 +83,60 @@ CommunicationStream::readMessage(Message &msg)
 #endif
    }
 
+void
+CommunicationStream::readMessageInFullCapacity(Message &msg)
```

Comment:
If the old `readMessage` is no longer used we can use the name for this function.

Reply:
I thought there might be the cases we'd still like to compare the performance between two types of read when debugging performance related issues down the road. So the old `readMessage` is kept for now.